pkg/model/resource/webhooks.go

```diff
@@ -77,6 +79,9 @@ func (webhooks *Webhooks) Update(other *Webhooks) error {
 	// Update conversion.
 	webhooks.Conversion = webhooks.Conversion || other.Conversion
 
+	// Update Spoke.
+	webhooks.Spoke = other.Spoke
+
```

Comment:
Since we are supporting only one `hub` - is there a way we can validate, where hub is not being changed. Or the spoke version was not previously declared as a `hub`. Or would this error occur while trying to scaffold the markers?

Reply:
I changed for []string
but maybe the right thing here would be have a List of GKV